doc/user/bgp.rst

```diff
@@ -2004,31 +2014,58 @@ Displaying BGP Information
 
          Total number of prefixes 1
 
-.. index:: show ip bgp regexp LINE
-.. clicmd:: show ip bgp regexp LINE
+Some other commands are available on that format, and permit to dump either IPv4
```

Comment:
`Some other commands provide additional options for filtering the output`, you already explained the v6 / v4 difference above

Reply:
ok